### ISSUE TITLE
Code for flashing 6507 ?

### ISSUE THREAD

**Issue by Maitresinh** [Dan-Grise/2600_clone_pcb#1]
Just a beotian question: i suppose we need to flash the 6507CPU. Any code and description of the process ?

**Comment by Dan-Grise** [comment]
I don't think you can flash the 6507.  My understanding is that it's a CPU
and not a microcontroller.  However I didn't really read it's datasheet.  I
used the original Atari 2600 schematic as a reference for the 6507
connections.  If it's possible to flash it I am unaware of any process.

Sorry.

**Comment by Maitresinh** [comment]
Thanks for the anwser. 
More than anything, my concern is : once build, do you need some kind of "bios" of anything like that ?

**Comment by Dan-Grise** [comment]
No.  Not for Atari 2600.  Coleco Vision and Intellivision require some sort
of BIOS.  The Atari 2600 does not.
